### DISCUSSION TITLE
How to download Sentinel1 precise orbit files as the present version is showing errors

### DISCUSSION THREAD

**Discussion by krsunil17** [isce-framework/isce2#822] [Q&A]
The Copernicus Scihub client has discontinued service from 31st October 2023 in favor of the new Copernicus Data Space Ecosystem. The new service no longer allows anonymous public downloads (using the gnssuser). So, how to download sentinel1 precise orbit files now using ISCE??

**Comment by fdelgadodelapuente** [comment]
From ASF

Precise orbits
https://s1qc.asf.alaska.edu/aux_poeorb/

Restituted orbits
https://s1qc.asf.alaska.edu/aux_resorb/

**Reply by krsunil17** [reply]
Thanks for the reply....how to automatically download these orbit files using isce?

**Reply by krsunil17** [reply]
Do we have to give credentials of earthdata or dataspace copernicus in the fetchOrbit.py file in ISCE2 environment?

**Comment by hardreddata** [comment]
I have been using https://github.com/scottstanie/sentineleof/tree/master with earthdata credentials.

Very handy!
